README.md

```diff
@@ -1,4 +1,6 @@
-# tgstation-server
+<div style= "text-align: center;">
+<img src =https://raw.githubusercontent.com/tgstation/tgstation-server/dev/build/logo.svg>
+</div>
```

Comment:
Try
```
<p align="center">
  <img src =./build/logo.svg>
</p>
```

Reply:
Will do! This makes sense as the file is already in the codebase!

Reply:
Hi Cyberboss, this change has been committed and pushed :)